src/simplewallet/simplewallet.cpp

```diff
@@ -3510,7 +3531,9 @@ bool simple_wallet::new_wallet(const boost::program_options::variables_map& vm,
     recovery_val = m_wallet->generate(m_wallet_file, std::move(rc.second).password(), recovery_key, recover, two_random, create_address_file);
     message_writer(console_color_white, true) << tr("Generated new wallet: ")
       << m_wallet->get_account().get_public_address_str(m_wallet->nettype());
-    std::cout << tr("View key: ") << string_tools::pod_to_hex(m_wallet->get_account().get_keys().m_view_secret_key) << ENDL;
+    std::cout << tr("View key: ");
```

Comment:
Looks like `std::flush` is necessary, otherwise the text output can get weird:
```
Generated new wallet: 543DAuRYixWJbnNZJRYekwjbi2KMGL79XDnDm1PN2LhiGRQtb6rJRSsjaJ87DtE43oYxVChQD3YyQiai2cQsPtw81zgoXDz
5fec71e61cc9a274afb8bf8596a7971a6357291dedc709b50a0c444f8626c407
View key: **********************************************************************
```